src/main/java/seedu/teachstack/model/person/UniquePersonList.java

```diff
@@ -82,6 +86,7 @@ public void remove(Person toRemove) {
     public void setPersons(UniquePersonList replacement) {
         requireNonNull(replacement);
         internalList.setAll(replacement.internalList);
+        sort();
```

Comment:
Would it be better to have sort students in display work through a command? Or is the default behavior you wanted.

Reply:
I suppose this is the default behavior